src/main/java/List.java

```diff
@@ -0,0 +1,32 @@
+public class List {
+    Task[] tasks = new Task[100];
+    private int tasksCount = 0;
+
+    public void addTask(String description) {
+        tasks[tasksCount] = new Task(description);
+        tasksCount++;
+        System.out.println("\t added: " + description);
+    }
+
+    public void listTasks() {
+        if (tasksCount == 0) {
+            System.out.println("\t You have no tasks added to the list.");
+            return;
+        }
+        for (int i = 0; i < tasksCount; i++) {
+            System.out.println("\t " + (i + 1) + ".[" + tasks[i].getStatusIcon() + "] " + tasks[i].getDescription());
```

Comment:
Perhaps improve the layout of this line? Using wrapped lines for lines with more than 120 chars could help improve the readability of this code!